ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Debug.scala

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.kvutils
+
+import java.io.{DataOutputStream, FileOutputStream}
+
+import com.daml.ledger.participant.state.kvutils.DamlKvutils._
+
+import scala.collection.JavaConverters._
+
+/** Utilities for debugging kvutils. */
+object Debug {
```

Comment:
Are you 100% certain that kvutils won't be used multiple times in the same process (e.g. some test)?

Reply:
99%, and 100% for this functionality.